app/src/marketplace/components/Modal/GetCryptoDialog/index.jsx

```diff
@@ -3,48 +3,66 @@
 import React from 'react'
 import { Translate, I18n } from 'react-redux-i18n'
 
-import WalletNoEthPng from '../../../assets/wallet_no_eth.png'
-import WalletNoEthPng2x from '../../../assets/wallet_no_eth@2x.png'
 import ModalPortal from '$shared/components/ModalPortal'
+import PngIcon from '$shared/components/PngIcon'
 import Dialog from '$shared/components/Dialog'
-import ExternalLinkButton from '$shared/components/Buttons/ExternalLinkButton'
+import Button from '$shared/components/Button'
+import Link from '$shared/components/Link'
+import { isMobile } from '$shared/utils/platform'
 
 import styles from './getCryptoDialog.pcss'
 
 export type Props = {
     onCancel: () => void,
 }
 
+const dialogText = () => {
+    if (isMobile()) {
+        return 'modal.getCryptoDialog.mobileMessage'
+    }
+    return 'modal.getCryptoDialog.message'
+}
+
 const GetCryptoDialog = ({ onCancel }: Props) => (
     <ModalPortal>
         <Dialog
             title={I18n.t('modal.getCryptoDialog.title')}
             onClose={onCancel}
         >
-            <img className={styles.icon} src={WalletNoEthPng} srcSet={`${WalletNoEthPng2x} 2x`} alt={I18n.t('error.wallet')} />
-            <Translate
-                value="modal.getCryptoDialog.message"
-                className={styles.message}
-                tag="p"
-                dangerousHTML
+            <PngIcon
+                className={styles.icon}
+                name="walletNoEth"
+                alt={I18n.t('modal.getCryptoDialog.title')}
             />
-
+            <Translate value={dialogText()} tag="p" dangerousHTML className={styles.message} />
```

Comment:
`value={dialogText()}` could be just:
```js
value={`modal.getCryptoDialog.${isMobile() ? 'mobileMessage' : 'message'}`}
```

Reply:
I did try something similar, and it didn't take for some reason, but this code of course works :D